build/Dockerfile.registry

```diff
@@ -1,10 +1,13 @@
-FROM quay.io/openshift/origin-operator-registry
+FROM quay.io/openshift/origin-operator-registry:4.2.0
 
-COPY manifests /registry
+COPY manifests manifests
```

Comment:
I'm not sure thus works OK with HCO/registry bundles. The filesystem layout is significant here.

Reply:
My source is just https://github.com/operator-framework/operator-registry

Reply:
according to this https://github.com/operator-framework/operator-registry/blob/master/cmd/registry-server/main.go#L36 registry server cares only about bundles.db path